tests/search.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { test, expect } from "@playwright/test";
+
+test("search", async ({ page }) => {
+  /** Reject the consent banner */
+  await page.goto("/");
+  await page.getByRole("button", { name: "Reject" }).click();
```

Comment:
Can we add an assertion that the heading of the cookie banner is visible? Just to make it clearer as to what we're rejecting.